package.json

```diff
@@ -10,7 +10,8 @@
     "lint:all": "nx run-many -t lint",
     "check-types": "nx affected -t typecheck",
     "check-types:all": "nx run-many -t typecheck",
-    "prepare": "[ -d .git ] && git config core.hooksPath '.git-hooks' || true"
+    "prepare": "[ -d .git ] && git config core.hooksPath '.git-hooks' || true",
+    "plan": "nx release plan"
```

Comment:
Should this be named `release-plan`? `plan` only doesn't say much about what it actually does.